docs/dev/ci/github_actions/runners.md

```diff
@@ -7,6 +7,9 @@ Two types of runners are available in this repository:
 * [GitHub Actions Runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners) - runners provided and managed by GitHub
 * [Self-hosted Runners](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) - runners created and managed by the OpenVINO CI team and linked to the OpenVINO repositories 
 
+Generally, it is advised to use the GitHub Actions runners for light jobs, like labelers, code style checks, etc, whereas
```

Comment:
```suggestion
Generally, it is advised to use the GitHub Actions runners for light jobs, like labellers, code style checks, etc., whereas
```

Reply:
Both these cases reflect the difference between American and British norms. I suggest to leave it the American way, for consistency.